src/plugins/validation/2and3/semantic-validators/schema-ibm.js

```diff
@@ -363,14 +363,15 @@ function checkEnumValues(schema, contextPath, config) {
 
   for (let i = 0; i < schema.enum.length; i++) {
     const enumValue = schema.enum[i];
-
-    const checkStatus = config.snake_case_only || 'off';
-    if (checkStatus.match('error|warning')) {
-      if (!isSnakecase(enumValue)) {
-        result[checkStatus].push({
-          path: contextPath.concat(['enum', i.toString()]),
-          message: 'Enum values must be lower snake case.'
-        });
+    if (typeof enumValue === 'string') {
+      const checkStatus = config.snake_case_only || 'off';
+      if (checkStatus.match('error|warning')) {
+        if (!isSnakecase(enumValue)) {
+          result[checkStatus].push({
+            path: contextPath.concat(['enum', i.toString()]),
+            message: 'Enum values must be lower snake case strings.'
```

Comment:
I don't think we need this "strings" qualifier added to the end of the message. 1) They don't _have_ to be strings, 2) the term "snake case" only applies to strings. Just adding the check is enough